english/TheTaleOfTheProgrammer

```diff
@@ -0,0 +1 @@
+once upon a time there was a random 14 year old kid who decided to learn git and github. little did he know that the pulling request bullshit would fuck his brain.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.